docs/changelog.md

```diff
@@ -1,5 +1,45 @@
 ## Changelog
 
+### v0.15.0 (1/7/2021)
+
+#### Highlights
+
```

Comment:
Need to add a section for compatibility to tell users why we need mmcv 1.3.8 following here https://github.com/open-mmlab/mmdetection/pull/5488/files